### DISCUSSION TITLE
Allow rearranging/pinning workflows in the Actions page.

### DISCUSSION THREAD

**Discussion by nirinchev** [community/community#50884] [Actions] [answered]
### Select Topic Area

Product Feedback

### Body

The left pane of the Actions page of a repository (e.g. https://github.com/realm/realm-dotnet/actions) shows all the workflows that have run in this repo. Especially with reusable workflows, the number of elements in that list can be fairly large and usually not all of them are interesting - for example if you want to manually trigger a `workflow_dispatch` workflow or find the results of a recent publish job. It would be a time saver if we could pin or otherwise rearrange the order in which workflows appear in the list so that the most useful are at the top.

(Attached image for visual aid, but hopefully the description is clear enough)

<img width="927" alt="Screenshot 2023-03-24 at 05 02 25" src="https://user-images.githubusercontent.com/2315687/227421698-48532a84-8eae-4ae7-942e-752821848844.png">

**Comment by cmyllis** [comment, marked as answer]
Seems like they added this now but the pins are global and not to each user preference :(

**Reply by nirinchev** [reply]
Sweet, I think that's still great for my use case, as I'd prefer all team members to immediately have access to the important workflows in my repo without having to reorder them themselves.
